src/control/server/instance_storage_rpc.go

```diff
@@ -198,7 +198,10 @@ func scanEngineBdevsOverDrpc(ctx context.Context, engine Engine, pbReq *ctlpb.Sc
 			return nil, errors.Errorf("smd %q has no ctrlr ref", sd.Uuid)
 		}
 
-		if sd.Ctrlr.DevState == ctlpb.NvmeDevState_UNPLUGGED {
+		if sd.Ctrlr.DevState != ctlpb.NvmeDevState_NORMAL &&
+			sd.Ctrlr.DevState != ctlpb.NvmeDevState_NEW &&
+			sd.Ctrlr.DevState != ctlpb.NvmeDevState_EVICTED {
```

Comment:
This will work, but if it were me I'd add a method to check this on `NvmeController`, e.g. `sd.Ctrlr.Scannable()`. That keeps the definition of what makes the controller scannable in one place, so it's easier to maintain when the inevitable future changes happen (e.g. to protobufs, etc).

Reply:
I can do this, probably should also do the same for the list of states that can be queried for health as that is different.